docs/source/rules.rst

```diff
@@ -38,7 +38,14 @@ ignore templating (``TMP``) & parsing (``PRS``) errors.
 
 .. code-block:: sql
 
-   WHERE dt >= DATE_ADD(CURRENT_DATE(), INTERVAL -2 DAY) -- noqa: PRS
+   WHERE
+     col1 = 2 AND
+     dt >= DATE_ADD(CURRENT_DATE(), INTERVAL -2 DAY) -- noqa: PRS
+
+.. note::
+   It should be noted that ignoring ``TMP`` and ``PRS`` errors can lead to
+   incorrect ``sqlfluff lint`` and ``sqfluff fix`` results as SQLFluff can
+   misinterpret the SQL being analysed.
```

Comment:
New note - @barrywhart might need to update this if we change `fix` to not fix.

Reply:
I added a comment on my issue about this: https://github.com/sqlfluff/sqlfluff/issues/2424#issuecomment-1028040994